cmd/scale.go

```diff
@@ -451,6 +446,19 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 	return f.SaveFile(sc.deploymentDirectory, "apimodel.json", b)
 }
 
+func vmInAgentPool(vmName string, sc *scaleCmd) bool {
+	if !strings.Contains(vmName, sc.agentPoolToScale) {
+		log.Warnf("VM %s is missing prefix \"%s\"", vmName, sc.agentPoolToScale)
+		return false
+	}
+	// Use strings.Contains for the nameSuffix as the Windows Agent Pools use only a substring of the first 5 characters of the entire nameSuffix
+	if !strings.Contains(vmName, sc.nameSuffix) {
```

Comment:
shouldn't this only check if the vmName contains the substring of the name suffix instead of the entire name suffix?